app/controllers/application_controller.rb

```diff
@@ -1155,7 +1155,11 @@ def get_view(db, options = {}, fetch_data = false)
 
     # Check for changed settings in params
     if params[:ppsetting] # User selected new per page value
-      @settings.store_path(:perpage, perpage_key(dbname), params[:ppsetting].to_i)
+      if dbname == "miqreportresult"
```

Comment:
Why is miqreportresult different?

Reply:
This just feels like we are missing an entry in perpage_key method for miqreportresult to return `:list`